docs/pages/config-plugins/plugins.mdx

```diff
@@ -133,7 +133,7 @@ export default withPlugin;
 
 ### Add TypeScript support and convert to dynamic app config
 
-We recommend writing config plugins in TypeScript, since this will provide intellisense for the configuration objects. However, your app config is ultimately evaluated by Node.js, which does not recognize TypeScript code by default. Therefore, you will need to add a parser for the TypeScript files from the the **plugins** directory to **app.config.ts** file.
+We recommend writing config plugins in TypeScript, since this will provide intellisense for the configuration objects. However, your app config is ultimately evaluated by Node.js, which does not recognize TypeScript code by default. Therefore, you will need to add a parser for the TypeScript files from the **plugins** directory to **app.config.ts** file.
```

Comment:
I think this is good as-is, just noting that node JS 24 can run TS by default, but probably not config plugins because we do use enums which are not supported syntax. Shouldn't be hard to get it running though.

Reply:
I don't think there's a need to track the ability to run on node 24 directly yet.
Thanks! :)